### ISSUE TITLE
Better display long Dataset names in the Table view

### ISSUE THREAD

**Issue by sseppi** [noi-techpark/it.bz.opendatahub.databrowser#603]

**Comment by pkritzinger** [comment]
@sseppi yes, truncating the name after having defined the max-length would be a great solution (do you know how long the dataset name can be?). Maybe on hover we can show the full dataset name by using the tooltip provided by the browser, s. example below.
![image](https://github.com/user-attachments/assets/1108dc9b-bb7f-4b1f-a6fa-a0fab57063f3)

**Comment by sseppi** [comment]
@pkritzinger we didn't define any limit for the dataset's name length. For now the "Echarging station accessibility" is the longest, but if future we could have even longer names.

I like the truncation of the name with the hover effect you proposed.

**Comment by gappc** [comment]
@sseppi @pkritzinger I recommend to let the browser do the truncation via css. Defining a max number of characters is not always helpful with non-monospaced fonts, e.g.

iiiiiiiiii => 10 chars
oooooooooo => 10 chars

**Comment by sseppi** [comment]
> @sseppi @pkritzinger I recommend to let the browser do the truncation via css. Defining a max number of characters is not always helpful with non-monospaced fonts, e.g.
> 
> iiiiiiiiii => 10 chars oooooooooo => 10 chars

@gappc Yes, you're right. I would do the truncation via css and implement the hover effect to see the full dataset name. Moreover, once the dropdown menu is open, in the list of dataset the full name must be shown, to let the user find the dataset he is looking for.

**Comment by pkritzinger** [comment]
@sseppi @gappc sounds good!
do you need any design specs from our side or can you directly implement it?

**Comment by sseppi** [comment]
@pkritzinger in my point of view is ready to be implemented.

**Comment by sseppi** [comment]
The feature seems to be implemented as expected
